src/main/groovy/org/mastercoin/test/TestSupport.groovy

```diff
@@ -265,7 +265,7 @@ trait TestSupport implements MastercoinClientDelegate {
      * @param amount     The number of units to create
      * @return The transaction hash
      */
-    Sha256Hash createProperty(Address address, Ecosystem ecosystem, PropertyType type, Long amount) {
+    Sha256Hash createProperty(Address address, Ecosystem ecosystem, PropertyType type, Number amount) {
```

Comment:
I think we want to encourage callers from using some `Number` types, such as `Float` or `Byte`. Maybe we want to go ahead an create a class to represent the MAX and MIN values for `NumberOfCoins`. Alternatively we should use a type that can represent all values (I think that would be `Long`) and throw exceptions if passed something outside the valid `MIN..MAX` range.

I'm thinking that some of these functions may be useful beyond `TestSupport` as a set of utilities for writing apps that use Omni Core. For creating an API we want the method signatures to be as strongly typed as possible to help eliminate errors at compile time. For testing, however, we might actually want a version of the routing that lets us send values that are too large.